endpoints-framework/src/main/java/com/google/api/server/spi/discovery/ProxyingDiscoveryService.java

```diff
@@ -37,7 +37,10 @@
  */
 @Api(
     name = "discovery",
-    version = "v1"
+    version = "v1",
+    title = "APIs Discovery Service",
+    description = "Lets you discover information about other Google APIs, such as what APIs are "
```

Comment:
This is verbatim the old discovery description, so I'd rather not change it. Plus, I think "lets" is more frequently used.